examples/models/llama2/export_llama_lib.py

```diff
@@ -225,6 +226,12 @@ def build_args_parser() -> argparse.ArgumentParser:
         default=f"{ckpt_dir}/params/demo_config.json",
         help="config.json",
     )
+    parser.add_argument(
```

Comment:
Also needs to specify QNN only, and be clear that we're just applying R1/R2 here.